package.json

```diff
@@ -27,18 +27,14 @@
       "eresource": "1.0",
       "license": "1.0",
       "location": "1.0",
-      "order_format": "1.0",
-      "order_type": "1.0",
       "payment_status": "1.0",
```

Comment:
payment_status is gone too, but the mod-orders-storage module descriptor incorrectly still has it listed.  @komick can you please fix the mod-orders-storage MD?